Documentation/project-docs/platform-names.md

```diff
@@ -56,6 +56,10 @@ Mono is an open source alternative to the .NET Framework. Mono started around th
 
 When .NET Core was released under the MIT license, Microsoft also released large chunks of the .NET Framework under the MIT license as well, which can be found [here](https://github.com/microsoft/referencesource). This enabled the Mono community to use the same code the .NET Framework uses in order to close gaps and avoid behavioral differences.
 
+Mono is primarily used to run .NET application on Linux and Mac OS X (though to get into the Mac App Store you need Xamarin, see below). There are ports of Mono to other platforms including PlayStation and Blackberry.
```

Comment:
~~application~~ applications

Mono runs on a lot more platforms, not sure if explicitly mentioning PlayStation/Blackberry is the way to go. Maybe we should just link to http://www.mono-project.com .